package.json

```diff
@@ -20,6 +20,7 @@
     "@ethersproject/strings": "^5.0.5",
     "@ethersproject/units": "^5.0.3",
     "@ethersproject/wallet": "^5.4.0",
+    "@snapshot-labs/snapshot.js": "^0.1.11",
```

Comment:
we are inside @snapshot-labs/snapshot.js... we don't need this.. you can remove it
```suggestion
```

Reply:
also, revert package-lock.json